newrelic.js

```diff
@@ -0,0 +1,70 @@
+var apiKey = require('./newrelicAPI');
```

Comment:
You can convert this to const as the apiKey won't be changing inside the newrelic.js file. 

**Rationale**: You aren't going to redeclare the variable apiKey and you don't want it to change inside of newrelic.js, only outside in your ./newrelicAPI file.